website/templates/Devices.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+{%block page_content %}
+<h1> Devices </h1>
+<p> A link in the room page (called manage devices) leads you to this page where you can:</p>
+
+    <a href="Device.html">Pair New Device</a>
+    <ul></ul>
+    <p1> All paired devices will appear here: </p1>
+       <ol>
+          <li><a href="Devices.html" >Device1</a> <ul>
+  <li> <a href="Devices.html"> Delete Paired Device</a></li>
+</ul> <ul>
+  <li> <a href="Devices.html">Add to a My Devices Group </a></li>
+</ul>
+          </li>
+               <li><a href="Triggers.html" >Device2</a></li>
+                      <li><a href="Triggers.html" >Device3</a></li>
+</ol>
+
+   <h1> Device Groups </h1>
+    <p>This page allows you to manage your Device groups. Device groups allow you to manage and control multiple devives together.</p>
+<style>
```

Comment:
You can put your CSS code in the styles block (outside the page_content block).
E.g.:
{%block styles%}
{{super()}}
table, td, th{...}
{%endblock%}